src/commands/ImportCommand.js

```diff
@@ -171,6 +171,11 @@ export default class ImportCommand extends Command {
       // Fall back to three-way merge, which can help with duplicate commits
       // due to merge history.
       ChildProcessUtilities.exec("git", ["am", "-3"], this.execOpts, (err) => {
+        if (err && /^patch is empty/i.test(err.stdout)) {
```

Comment:
This was already fixed in https://github.com/lerna/lerna/pull/1059/files#diff-84077e6291e8b6b2a596aff7f258b0dd, but the PR view doesn't seem to incorporate those changes.

I like your unit test, but you need to rebase and remove this block before I can merge it.